po/fr.po

```diff
@@ -72,7 +72,7 @@ msgstr "Montrer icône du panneau"
 
 #: src/Widgets/Headerbar.vala:62
 msgid "Start in background:"
-msgstr ""
+msgstr "Démarrer en arrière-plan"
```

Comment:
I don't understand French, but don't you need the trailing colon?

Reply:
Oups, my bad !